content/rule_book/quiz_process.md

```diff
@@ -1,106 +1,173 @@
 - If a quizzer's light comes on after the quizmaster has discernibly begun reading the question, this will be considered a valid jump, and the quizzer will be given an opportunity to answer.
-- The first quizzer to rise and trigger their light, provided there has been no foul, will be awarded the jump.
+- The 1st quizzer to rise and trigger their light, provided there has been no foul, will be awarded the jump.
 - If in the judgment of the officials, a deliberate attempt is made to forfeit a question, an error will be charged, and 10 points deducted from the team score. The next question will be the same question number.
 
-### Tie Jumping
+## Timeliness
 
-- When members of the same team tie, the captain will select one quizzer to answer without any type of verbal or non-verbal communication.
-- When quizzers of two or three teams tie, the question will be replaced with a question of the same type on which all eligible members of the tied teams may jump. The question number remains the same.
-- When a quizzer errs on the tie-jump question, it will be treated as a regular question and the quiz will continue as normal.
+Except for the exception below:
 
-## Time Limits
+- Quizzes will start at the scheduled or announced time.
+- Any team that is late does not receive the starting 20 points awarded for being on time.
+- Any team more than 10 minutes late forfeits the quiz.
 
-- Quizzes will start at the scheduled or announced time. Any team that is late does not receive the starting 20 points awarded for being on time.
-    - If the late team is currently quizzing in another location which is running over the time limit, they are not required to forfeit the quiz or lose 20 points.
-    - Any team more than 10 minutes late (after the scheduled or announced time for the quiz) forfeits the quiz.
-- If an individual quizzer is late, the quiz will start on time. The coach has the option to put the late quizzer in during a timeout, but the team will forfeit the 20 points the team starts with. The coach also has the option to keep the quizzer out of the quiz and keep the 20 points but must notify the quizmaster at the beginning of the quiz, of their decision.
-- Quizzers have five seconds in which to jump after the question is completed.
+The exception to the above is if the late team was late due to quizzing in another location which is running late, they are not required to forfeit the quiz or lose 20 points.
 
-A quizzer has 30 seconds to complete their answer after being recognized.
+If an individual quizzer is late, the quiz will start on time. The coach has the option to put the late quizzer in during a timeout, but the team will forfeit the 20 points the team starts with. The coach also has the option to keep the quizzer out of the quiz and keep the 20 points but must notify the quizmaster at the beginning of the quiz of their decision.
```

Comment:
"if any number of members of a team is late...."

Reply:
bb34875